python/tvm/hybrid/util.py

```diff
@@ -86,3 +87,20 @@ def _restore_runtime(func, intersect):
         _globals.pop(elem)
     for k, v in intersect:
         _globals[k] = v
+
+
+def replace_io(body, rmap):
```

Comment:
what's this function for, I don't see any caller in this PR.

Reply:
In `visit_Assign`.
I think this is to resolve the same issue I mentioned in HybridOpNode.
You asked me to talk that story, I hope you still remember.

Reply:
Yes, I remember